app/uk/gov/hmrc/cdsimportsddsfrontend/domain/DocumentationType.scala

```diff
@@ -16,41 +16,57 @@
 
 package uk.gov.hmrc.cdsimportsddsfrontend.domain
 
+case class WriteOffType(quantity: Option[String])
+
+case class AdditionalDocumentType(
+  categoryCode: Option[String],
+  typeCode: Option[String],
+  id: Option[String],
+  lpco: Option[String],
+  name: Option[String],
+  effectiveDateTime: Option[String],
+  submitter: Option[String],
+  writeOff: Option[String]
+)
+
+case class AdditionalPaymentType(
+  additionalDocPaymentID: Option[String],
+  additionalDocPaymentCategory: Option[String],
+  additionalDocPaymentType: Option[String]
+)
+
 case class DocumentationType(
   previousDocCategory:  Option[String],
   previousDocType: Option[String],
   previousDocReference: Option[String],
   previousDocGoodsItemId: Option[String],
   additionalInfoCode: Option[String],
   additionalInfoDescription: Option[String],
-  additionalDocCategoryCode: Option[String],
-  additionalDocTypeCode: Option[String],
-  additionalDocId: Option[String],
-  additionalDocLPCO: Option[String],
-  additionalDocName: Option[String],
+  additionalDocument: Seq[AdditionalDocumentType],
   localReferenceNumber: Option[String],
-  additionalDocPaymentID: Option[String],
-  additionalDocPaymentCategory: Option[String],
-  additionalDocPaymentType: Option[String]
+  additionalPayment: Seq[AdditionalPaymentType]
 )
 
 object DocumentationType {
```

Comment:
The name shouldn't include the word `Type`

The CDS PDF name for section 2 is References so this object should be called References

Reply:
Agree, but this follows the existing pattern. This can be done as part of a refactor after this ticket.